src/daemon/processor/mod.rs

```diff
@@ -67,7 +71,7 @@ impl ProcessorRequest {
         self.dimensions
     }
 
-    fn split(self) -> (Vec<String>, Transition, Option<GifProcessor>) {
+    fn split(self) -> (Vec<String>, Transition, Option<GifProcessor>,(usize,usize),usize){
```

Comment:
The center position and acceleration are properties of the transition, and so they should be inside the `Transition` struct, not outside it. By putting these things inside `Transition`, many of the function signatures you changed can actually remain exactly as they were before, unchanged. It will greatly simplify the implementation of this functionality, I think.